extension/src/popup/components/sendPayment/SendTo/index.tsx

```diff
@@ -225,6 +225,7 @@ export const SendTo = ({ previous }: { previous: ROUTES }) => {
                       <button
                         onClick={async () => {
                           setIsLoading(true);
+                          setValidatedPubKey(pubKey);
```

Comment:
this fixes an issue where if you clicked a recent address, it would say the account was unfunded because it didn't know it needed to lock up the destination account's balance on Ln 186